climada/hazard/test/test_trop_cyclone.py

```diff
@@ -306,20 +306,42 @@ def test_holland_2010_pass(self):
         """Test Holland et al. 2010 wind field model."""
         # test at centroids within and outside of radius of max wind
         si_track = xr.Dataset({
-            "rad": ("time", KM_TO_M * np.array([75, 40])),
-            "vmax": ("time", [35.0, 40.0]),
-            "hol_b": ("time", [1.80, 2.5]),
+            "rad": ("time", KM_TO_M * np.array([75, 40, 70, 103, 96])),
+            "vmax": ("time", [35.0, 40.0, 90.0, 18.0, 12.0]),
+            "hol_b": ("time", [1.80, 2.5, 2.5, 1.9, 1.75]),
         })
-        d_centr = KM_TO_M * np.array([[35, 75, 220], [30, 1000, 300]], dtype=float)
-        close_centr = np.array([[True, True, True], [True, False, True]], dtype=bool)
+        d_centr = KM_TO_M * np.array([
+            [35, 75, 220],
+            [30, 1000, 300],
+            [35, 105, 300],
+            [35, 105, 300],
+            [35, 105, 300],
+        ], dtype=float)
+        close_centr = np.array([
+            [True, True, True],
+            [True, False, True],
+            [True, True, True],
+            [True, True, True],
+            [True, True, True],
+        ], dtype=bool)
         hol_x = _x_holland_2010(si_track, d_centr, close_centr)
-        np.testing.assert_array_almost_equal(
-            hol_x, [[0.5, 0.5, 0.47273], [0.5, 0, 0.211602]])
+        np.testing.assert_array_almost_equal(hol_x, [
+            [0.5, 0.500000, 0.472730],
+            [0.5, 0.000000, 0.211602],
+            [0.5, 0.519094, 0.625478],
+            [0.5, 0.495423, 0.049174],
+            [0.5, 0.464344, 0.000000],
```

Comment:
I am guessing the two zeros here are an effect of the clipping an something you wanted to explicitly test for?

Reply:
In my understanding, the first zero results from the large distance between track and centroids (`close_centr = False`), thus does not test the clipping. The last zero tests the clipping by creating the conflict v_max_s < v_n (v_max_s = 12 and v_n defaulting to 17). To me, it is also unclear what is tested in the other two, added cases.